docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -2959,6 +2959,11 @@ The `memberlist` block configures the Gossip memberlist.
 # CLI flag: -memberlist.compression-enabled
 [compression_enabled: <boolean> | default = true]
 
+# (advanced) How frequently to notify watchers when a key changes. Can reduce
+# CPU activity in large memberlist deployments. 0 to notify without delay.
```

Comment:
```suggestion
# CPU activity in large member list deployments. Set to `0` to notify without delay.
```

Reply:
Thank you- I’ll make these changes on the main branch. If I make them here no one will ever see them. 😺